packages/experiments/src/components/Chiclet/ChicletCard.tsx

```diff
@@ -0,0 +1,126 @@
+import * as React from 'react';
+import {
+  BaseComponent,
+  css
+} from '../../Utilities';
+import {
+  IChicletCardStyles
+} from './ChicletCard.types';
+import { IChicletCardProps, IChicletAction } from './ChicletCard.types';
+import { mergeStyles } from '../../Styling';
+import {
+  getClassNames
+} from './Chiclet.styles';
+import { Icon } from 'office-ui-fabric-react/lib/Icon';
+import { IconButton, IButtonProps } from 'office-ui-fabric-react/lib/Button';
+import { Image } from 'office-ui-fabric-react/lib/Image';
+import { TestImages } from 'office-ui-fabric-react/src/common/TestImages';
+
+export class ChicletCard extends BaseComponent<IChicletCardProps, any> {
+  private _classNames: IChicletCardStyles = {};
+
+  public render() {
+    const { styles: customStyles, title, ogType, description, image, imageType, imageWidth, imageHeight, imageAlt, url, onClick, className, actions, theme } = this.props;
+    const actionable = (onClick) ? true : false;
+
+    this._classNames = getClassNames(theme!, customStyles);
+
+    // if this element is actionable it should have an aria role
+    const role = actionable ? (onClick ? 'button' : 'link') : undefined;
+    const tabIndex = actionable ? 0 : undefined;
+
+    var preview = this._renderPreviewImage(image, imageHeight, imageWidth, ogType);
+
+    return (
+      <div
+        tabIndex={ tabIndex }
+        role={ role }
+        onClick={ actionable ? this._onClick : undefined }
+        className={
+          css('ms-ChicletCard', className, mergeStyles(this._classNames.root)) }
```

Comment:
Instead of using the `css` helper, create a `Chiclet.styles.ts` file and do your styling logic there.